modules/olm-creating-catalog-from-index.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * operators/managing-custom-catalogs.adoc
+
+[id="olm-creating-catalog-from-index_{context}"]
+= Creating a catalog from an index image
+
+To create a catalog from an index image and apply it to a {product-title}
+cluster:
+
+.Prerequisites
+
+* `opm` version 1.12.1
```

Comment:
We're up to 1.12.7 as of today

Reply:
Actually, if you already have an index image I don't know that you need opm for this exercise.

Reply:
Oops yeah, got copy/paste happy.

Looks like I was still using the opm from the downstream 4.4 image, which didn't have `opm version` yet... When I grab the one from the upstream 4.5.0 image, I can verify with `opm version` that it's 1.12.3, so I'll put 1.12.3+ in the remaining spots that mention opm's version requirement.